packages/docusaurus-theme-classic/src/theme/CodeBlock/index.tsx

```diff
@@ -18,52 +18,63 @@ import styles from './styles.module.css';
 
 import {useThemeConfig, parseCodeBlockTitle} from '@docusaurus/theme-common';
 
-const highlightLinesRangeRegex = /{([\d,-]+)}/;
-const languageTypes = ['js', 'jsBlock', 'jsx', 'python', 'html'] as const;
+const HighlightLinesRangeRegex = /{([\d,-]+)}/;
```

Comment:
As I understand PascalCased variables should be objects right? (Also noticed this in `docusaurus-init` where `TypeScriptTemplateSuffix` is PascalCased)

Reply:
I don't have a strong opinion about JS constant namings as long as it's clear enough those are constants. We already have this convention in other places so just made it consistent, but we can also redefine those conventions if needed